stubs/lzstring/lzstring/__init__.pyi

```diff
@@ -0,0 +1,17 @@
+class LZString:
+    @staticmethod
+    def compress(uncompressed: str) -> str: ...
+    @staticmethod
+    def compressToUTF16(uncompressed: str) -> str: ...
+    @staticmethod
+    def compressToBase64(uncompressed: str) -> str: ...
+    @staticmethod
+    def compressToEncodedURIComponent(uncompressed: str) -> str: ...
+    @staticmethod
+    def decompress(compressed: str) -> str: ...
+    @staticmethod
+    def decompressFromUTF16(compressed: str) -> str: ...
+    @staticmethod
+    def decompressFromBase64(compressed: str) -> str: ...
+    @staticmethod
+    def decompressFromEncodedURIComponent(compressed: str) -> str: ...
```

Comment:
The runtime implementations for all of these seem to explicitly check to see if `None` was passed in; maybe we should allow that? https://github.com/gkovacs/lz-string-python/blob/916183026bdc80c33913f97b4927bd20df109987/lzstring/__init__.py#L371-L430

```suggestion
    def compress(uncompressed: str | None) -> str: ...
    @staticmethod
    def compressToUTF16(uncompressed: str | None) -> str: ...
    @staticmethod
    def compressToBase64(uncompressed: str | None) -> str: ...
    @staticmethod
    def compressToEncodedURIComponent(uncompressed: str | None) -> str: ...
    @staticmethod
    def decompress(compressed: str | None) -> str: ...
    @staticmethod
    def decompressFromUTF16(compressed: str | None) -> str: ...
    @staticmethod
    def decompressFromBase64(compressed: str | None) -> str: ...
    @staticmethod
    def decompressFromEncodedURIComponent(compressed: str | None) -> str: ...
```

Reply:
The decompress methods can also return None:

```
>>> LZString.decompress('foo bar') is None
True
```